Tests/conf.json

```diff
@@ -5565,6 +5575,9 @@
             "integrations": "Forcepoint Security Management Center",
             "playbookID": "ForcepointSecurityManagementCenter_test",
             "is_mockable": false
+        },
+        {
+            "playbookID": "0d7887b1-0266-407b-870a-f7b3e27fdccb"
```

Comment:
Wasn't able to find this ID, is this a new playbook? we usually have the identical id and name